readthedocs/rtd_tests/tests/test_core_tags.py

```diff
@@ -9,14 +9,12 @@
 from readthedocs.projects.models import Project
 
 
-@pytest.mark.usefixtures("url_scheme")
 @override_settings(USE_SUBDOMAIN=False, PRODUCTION_DOMAIN='readthedocs.org')
 class CoreTagsTests(TestCase):
     fixtures = ['eric', 'test_data']
 
     def setUp(self):
-        url_base = '{scheme}://{domain}/docs/pip{{version}}'.format(
-            scheme=self.url_scheme,
+        url_base = 'http://{domain}/docs/pip{{version}}'.format(
```

Comment:
Why don't respect `PUBLIC_DOMAIN_USES_HTTPS` setting here as well?

Reply:
Tests aren't using a subdomain, `PUBLIC_DOMAIN` and `PUBLIC_DOMAIN_USES_HTTPS` only work with `USE_SUBDOMAIN=True`